### DISCUSSION TITLE
Newest versions v2.5* won't open on mac M1

### DISCUSSION THREAD

**Discussion by ocochodzi111** [supermerill/SuperSlicer#3049] [Q&A] [answered]
v2.4 working ok, all newest versions (v2.5* intel macos) gives error popup on opening -> internal error: basic_string. It shows up on loading screen (when "loading setting tab" comment appearing - or something like that - translated from my native language). No crash screen but app close itself.
Anyone knows something about it?

**Comment by supermerill** [comment, marked as answer]
already reported.
I don't have an intel mac to debug that.
Someone with one and some dev background can help me fix it?

**Reply by supermerill** [reply]
fixed #3038
